pmd-core/src/main/java/net/sourceforge/pmd/cpd/XMLRenderer.java

```diff
@@ -32,6 +36,8 @@
  *
  */
 public final class XMLRenderer implements CPDReportRenderer {
+    private static final String NAMESPACE_URI = "https://pmd-code.org/ns/cpd-report/1.0.0";
+    private static final String NAMESPACE_LOCATION = "https://pmd-code.org/ns/cpd-report_1_0_0.xsd";
```

Comment:
I don't recall…

1. Is this the same location were are are publishing the other XML Schemas such as PMD ruleset and PMD report?
2. How is the schema being published there?

Reply:
It's not the same location. The old location for PMD's report is here:
https://github.com/pmd/pmd/blob/c2dc7b3d9b1aa93554dff8e0443a5508a08c7c61/pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java#L46

Which should have been updated at some time to https (as redirects are not always followed by all tools...). But that's something else.

For the new schema, I've now settled for https://pmd.github.io/schema/cpd-report_1_0_0.xsd - which means, we should push this file then to https://github.com/pmd/pmd.github.io (and just to be consistent also to https://github.com/pmd/pmd.soureforge.net). Eventually, we probably should streamline the 3 web spaces into one -> pmd-code.org. But again, that's something else.
The actual update of http://pmd.sourceforge.net (or https://pmd.sourceforge.io) is done via rsync locally (not automated, if I recall correctly).